### DISCUSSION TITLE
Add a timer when sending mails for confirming your account

### DISCUSSION THREAD

**Discussion by lejdiprifti** [keycloak/keycloak#24545] [Ideas]
In order to avoid spamming and overloading of STMP servers and malicious use, why don't we add the possibility to add a timer when sending mails for confirming the account?

**Comment by MariiaAlekseichuk** [comment]
I also interested in this question.
It would be great to store last time of email sending in Infinispan cache without database persisting.
Any ideas about the way to implement this?

**Comment by ZaibanAli** [comment]
Is there any progress on this one?
